sources/osgAnimation/RigTransformHardware.js

```diff
@@ -15,84 +19,74 @@ define( [
     function getShader( nbVec4 ) {
```

Comment:
is for debug purpose ? 
To remove when  validated the remove of the "timeout" thingy
(as I couldn't reproduce here the bug that made that timeout necessary)